projects/components/src/table/controls/table-controls.component.ts

```diff
@@ -161,7 +161,7 @@ export class TableControlsComponent implements OnChanges {
     this.checkboxDiffer = this.differFactory.find([]).create();
 
     this.subscriptionLifecycle.add(
-      this.searchDebounceSubject.pipe(debounceTime(200)).subscribe(text => this.searchChange.emit(text))
```

Comment:
We can really use the ootb debounceTime property of search box component. Additional subscription here is not required in this table.

Please update the time in search box component as well if this looks good,

Reply:
ah yeah I forgot we added that - switched over to using that one (searchbox doesn't have a built in time, it's not debouncing by default)